### DISCUSSION TITLE
Exclude Accessories

### DISCUSSION THREAD

**Discussion by therealbrad** [aarons22/homebridge-bond#224] [Q&A]
I know i can exclude the dimmer and Toggle State accessories, but my fan has the option of both an up light and down light. I only have the down light but I get accessories for both even though I removed the commands for the up light in the Bond App. Now when I say "turn off the bedroom lights", it issues both commands which toggles the light off and then on again. So my question is.. can I disable one of the light accessories?

**Comment by passionatelogic** [comment]
I'm noticing this too -- removing a "command" from the Bond App doesn't remove it from HOOBS/Homebridge. I'd love to see only ONE accessory show up in HomeKit for my shades instead of the extra "Preset" toggle that are added to Bond by default for Somfy blinds.
